treeherder/model/management/commands/test_analyze_perf.py

```diff
@@ -74,10 +74,15 @@ def handle(self, *args, **options):
                 signature_data = pc.get_performance_signatures(
                     project, time_interval=options['time_interval'])
                 signatures = []
+                signatures_should_ignored = []
                 # if doing everything, only handle summary series
                 for (signature, properties) in signature_data.iteritems():
+                    signatures.append(signature)
                     if 'subtest_signatures' in properties:
-                        signatures.append(signature)
+                        signatures_should_ignored = signatures_should_ignored + \
```

Comment:
Also we should have a comment here like '# Don't alert on subtests which have a summary'